pkg/setting/setting.go

```diff
@@ -815,10 +817,14 @@ func (cfg *Cfg) Load(args *CommandLineArgs) error {
 	cfg.MetricsEndpointDisableTotalStats = iniFile.Section("metrics").Key("disable_total_stats").MustBool(false)
 
 	analytics := iniFile.Section("analytics")
-	ReportingEnabled = analytics.Key("reporting_enabled").MustBool(true)
 	CheckForUpdates = analytics.Key("check_for_updates").MustBool(true)
 	GoogleAnalyticsId = analytics.Key("google_analytics_ua_id").String()
 	GoogleTagManagerId = analytics.Key("google_tag_manager_id").String()
+	ReportingEnabled = analytics.Key("reporting_enabled").MustBool(true)
+	reportingDistributor := analytics.Key("reporting_distributor").String()
+	if len(reportingDistributor) >= 100 {
+		ReportingDistributor = reportingDistributor[:100]
+	}
```

Comment:
ReportingDistributor isn't set here unless the length is >= 100

Reply:
:facepalm: